src/test/resources/feature/5-helpdesk/001_074_helpDeskSezioneRicerca.feature

```diff
@@ -10,5 +10,5 @@ Feature: utente helpdesk visualizza pagina sezione ricerca ed estrazione dati
 
   Scenario: visualizzazione corretta pagina sezione ricerca ed estrazione dati in helpdesk
     When Nella Home di helpdesk utente clicca su sezione ricerca ed estrazione dati
-    And visualizzazione corrett pagina ricerca ed estrazione dati
+    And visualizzazione corretta pagina ricerca ed estrazione dati
```

Comment:
manca il Then. cambia And nell'ultimo step

Reply:
done